src/App.js

```diff
@@ -26,6 +27,7 @@ const App = () => {
                 }
               >
                 <Route path="/welcome" element={<Welcome />} />
+                <Route path="/homeEmptyState" element={<HomeEmptyState />} />
```

Comment:
I'd recommend to create a constant to store your different routes and use it wherever you need routes

Reply:
Will do this and other improvements on a separate PR, thanks for the suggestion!